en/docs/reference/mediators/property-reference/generic-properties.md

```diff
@@ -477,11 +477,11 @@ Generic properties allow you to configure messages as they're processed by the 
 </tr>
 <tr class="odd">
 <td><p><strong>Description</strong></p></td>
-<td><p>If this property is set to <code>              true             </code> , the result of file content processing carried out by the <a href="{{base_path}}/reference/mediators/smooks-Mediator">Smooks Mediator</a> will not be loaded into the message context. This is useful in situations where you want to avoid large memory growth/out of heap space issue that may occur when large files processed by the Smooks mediator are <reprocessed class=""></reprocessed></p></td>
+<td><p>If this property is set to <code>              true             </code> , the result of file content processing carried out by the <a href="{{base_path}}/reference/mediators/smooks-Mediator">Smooks Mediator</a> will not be loaded into the message context. This is useful in situations where you want to avoid large memory growth/out of heap space issue that may occur when large files processed by the Smooks mediator are</p></td>
```

Comment:
`<td><p>If this property is set to <code>              true             </code> , the result of file content processing carried out by the <a href="{{base_path}}/reference/mediators/smooks-Mediator">Smooks Mediator</a> will not be loaded into the message context. This is useful in situations where you want to avoid large memory growth/out of heap space issue that may occur when large files processed by the Smooks mediator are reprocessed.</p></td>`